2000-09-04  Craig Newell  <CraigN@ieee.org>

	* gcc.c: Undefine "__WCHAR_TYPE__" before redefining it.

--- gcc/gcc.c.jj	Thu Dec 28 13:25:17 2000
+++ gcc/gcc.c	Thu Dec 28 13:25:42 2000
@@ -559,7 +559,7 @@ static const char *cpp_options =
  %{!undef:%{!ansi:%{!std=*:%p}%{std=gnu*:%p}} %P} %{trigraphs}\
  %c %{Os:-D__OPTIMIZE_SIZE__} %{O*:%{!O0:-D__OPTIMIZE__}}\
  %{ffast-math:-D__FAST_MATH__}\
- %{fshort-wchar:-D__WCHAR_TYPE__=short\\ unsigned\\ int}\
+ %{fshort-wchar:-U__WCHAR_TYPE__ -D__WCHAR_TYPE__=short\\ unsigned\\ int}\
  %{fshow-column} %{fno-show-column}\
  %{fleading-underscore} %{fno-leading-underscore}\
  %{foperator-names} %{fno-operator-names}\
